### DISCUSSION TITLE
Status value in List view should reflect if a document has had changes since being published

### DISCUSSION THREAD

**Discussion by tiago-meireles** [payloadcms/payload#7608] [Feature Requests & Ideas]
### Payload Version

beta.76

### Node Version

20 LTS

### Next.js Version

15.0.0-canary.104

### Describe the Bug

When the draft versions and autosave options are enabled, the (published) status that's displayed in the Collection List do not match the status that's displayed when the entry is selected.

![image](https://github.com/user-attachments/assets/d35f9915-fce1-4b45-a573-f28e990d09c9)

![image](https://github.com/user-attachments/assets/b283155b-8e49-47d1-b956-1efd47aa6af8)

- For example, the page list shows the status of the page as `Draft` while the status of the page is `Changed` when selected.
- This happens when an entry is published and the content is updated (and saved). 
- This does not affect the page API results, but it makes it more difficult to identify in the Admin if a page is a draft or a published page that had changes to the content.

### Reproduction Steps

- Created a project with the `npx create-payload-app@beta` command
- Update it to the latest Payload release.
- Created a page and published it.
- Check the status that is displayed in the page collection list (it should be `Published`)
- Change the page title (the status of that page should be `Changed`)
- Check the status that is displayed in the page collection list (it should be `Draft`)

**Comment by paulpopus** [comment]
This is a tough one to solve without making additional fetches. The view list fetches data with `draft: true` and as such we can't know at that point if there's also a published version.

We will be addressing this but I've moved it to a feature request as it's closely related to the ability to [publish individual locales](https://github.com/payloadcms/payload/discussions/4194), internally.

**Comment by pranavks** [comment]
We need this too.

Our content team is having difficulties distinguishing Published vs Draft posts from the list view. We'd like the list view status of the Changed posts to be something like 'Changed'.
